.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+# See https://pre-commit.com for more information
```

Comment:
We can discuss how useful it is to have pre-commit as we mainly have Rust files, but if we have it we need this file.
If not we should remove the corresponding workflow.